ZFitter/script/bash_functions_calibration.sh

```diff
@@ -20,7 +20,8 @@ pileupTrees(){
 for tag in `grep "^s" data/validation/${file}.dat | grep selected | awk -F" " ' { print $1 } '`
 do
     #tag is s1, s2, ...
-    mv tmp/mcPUtree${tag}.root  ${eos_path}/data/puTree/${tag}_${file}.root
+    mv tmp/mcPUtree${tag}.root ${eos_path}/data/puTree/${tag}_${file}.root
+    #mv tmp/${tag}_pileup_chain.root  ${eos_path}/data/puTree/${tag}_${file}.root #this does not work
```

Comment:
Ok. At the moment it's there to remind me not to use tmp/${tag}_pileup_chain.root